packages/gatsby/src/utils/get-server-data.ts

```diff
@@ -5,7 +5,7 @@ import { match } from "@gatsbyjs/reach-router/lib/utils"
 
 export interface IServerData {
   headers?: Record<string, string>
-  props?: Record<string, unknown>
+  props?: Record<string, string | number | Array<string>>
```

Comment:
I would keep this unknown

Reply:
Right, also this wasn't place I want to change (I wanted to change `headers` not `props`), but given previous comment I'll probably just revert this

Reply:
I'm okay with the above. Node will take care of it